nautobot/docs/release-notes/version-1.3.md

```diff
@@ -0,0 +1,231 @@
+- A new set of REST API endpoints have been added to `/api/extras/jobs/<uuid>/`. The existing `/api/extras/jobs/<class_path>/` REST API endpoints continue to work but should be considered as deprecated.
+    - A new version of the REST API `/api/extras/jobs/` list endpoint has been implemented as well, but by default this endpoint continues to demonstrate the pre-1.3 behavior unless the REST API client explicitly requests API `version=1.3`. See the section on REST API versioning, below, for more details.
+- As a minor security measure, newly installed Jobs default to `enabled = False`, preventing them from being run until an administrator or user with appropriate permissions updates them to be enabled for running.
+
+!!! note
+    As a convenience measure, when initially upgrading to Nautobot 1.3.x, any existing Jobs that have been run or scheduled previously (i.e., have at least one associated JobResult and/or ScheduledJob record) will instead default to `enabled = True` so that they may continue to be run without requiring changes.
+
+For more details please refer to the [Jobs feature documentation](../additional-features/jobs.md) as well as the [Job data model documentation](../models/extras/job.md).
+
+#### JSON Type for Custom Fields ([#897](https://github.com/nautobot/nautobot/issues/897))
+
+Custom fields can now have a type of "json". Fields of this type can be used to store arbitrary JSON data.
+
+#### Natural Indexing for Common Lookups ([#1638](https://github.com/nautobot/nautobot/issues/1638))
+
+Many fields have had indexing added to them as well as index togethers on `ObjectChange` fields. This should provide a noticeable performance improvement when filtering and doing lookups.
+
+!!! note
+    This is going to perform several migrations to add all of the indexes. On MySQL databases and tables with 1M+ records this can take a few minutes. Every environment is different but it should be expected for this upgrade to take some time.
+
+#### Overlapping/Multiple NAT Support ([#630](https://github.com/nautobot/nautobot/issues/630))
+
+IP addresses can now be associated with multiple outside NAT IP addresses. To do this, set more than one IP Address to have the same NAT inside IP address.
+
+A new version of the REST API `/api/ipam/ip-addresses/*` endpoints have been implemented as well, but by default this endpoint continues to demonstrate the pre-1.3 behavior unless the REST API client explicitly requests API `version=1.3`. See the section on REST API versioning, below, for more details.
+
+!!! note
+    There are some guardrails on this feature to support backwards compatibility. If you consume the API without specifying the version header or query argument and start associating multiple IPs to have the same NAT inside IP address, an error will be thrown. Existing schema returns `nat_outside` as a single object, where as 1.3 and beyond will return this as a list.
```

Comment:
```suggestion
    There are some guardrails on this feature to support backwards compatibility. If you consume the REST API without specifying the version header or query argument and start associating multiple IPs to have the same NAT inside IP address, an error will be reported, because the existing REST API schema returns `nat_outside` as a single object, where as 1.3 and beyond will return this as a list.
```